docs/multicluster/architecture.md

```diff
@@ -34,33 +36,42 @@ given ClusterSet.
 
 ## Antrea Multi-cluster Controller
 
-In a member cluster, Antrea Multi-cluster creates a Deployment that runs Antrea Multi-cluster
-Controller which is responsible for exporting resource to and importing resource from a leader
-cluster in a ClusterSet.
+Antrea Multi-cluster Controller watches ClusterSet, ClutserClaim, Service, ServiceExport etc.
```

Comment:
Remove "watches...". It is not more important than other responsibilities of MC Controller to mention first.

Reply:
If you want, we can say "Antrea Multi-cluster Controller implements ClusterSet management and resource export/import in the ClusterSet".